govcd/vdc_test.go

```diff
@@ -145,6 +145,26 @@ func (vcd *TestVCD) Test_NewVdc(check *C) {
 
 }
 
+// Test_GetVDCHardwareVersion tests hardware version fetching functionality
+func (vcd *TestVCD) Test_GetVDCHardwareVersion(check *C) {
+	err := vcd.vdc.Refresh()
+	check.Assert(err, IsNil)
+
+	// vmx-18 is the latest version supported by 10.3.0, the oldest version we support.
```

Comment:
Does this assertion stand after we drop support for 10.3? Will it still run reliably when the lowest supported version changes to 10.4, 10.5?

Reply:
10.3, 10.4 and 10.5 all support this version, 10.4 and 10.5 also support `vmx-19` but I just kept it at `vmx-18`